src/lib/components/BucketAggregation/BucketAggregation.js

```diff
@@ -85,13 +85,34 @@ BucketAggregation.propTypes = {
   userSelectionFilters: PropTypes.array.isRequired,
   resultsAggregations: PropTypes.object.isRequired,
   updateQueryFilters: PropTypes.func.isRequired,
-  renderElement: PropTypes.func,
   renderValuesContainerElement: PropTypes.func,
   renderValueElement: PropTypes.func,
+  overridableUID: PropTypes.string,
 };
 
 BucketAggregation.defaultProps = {
-  renderElement: null,
   renderValuesContainerElement: null,
   renderValueElement: null,
+  overridableUID: '',
+};
+
+const Element = (props) => {
+  const { title, containerCmp, overridableUID } = props;
+  return (
+    containerCmp && (
+      <Overridable
+        id={buildUID('BucketAggregation.element', overridableUID)}
+        {...props}
```

Comment:
will it pass more `props` here? Shall we have a strict interface instead so that when you override you get only what you can use?
e.g.
```
const myNewComp = ({ title, containerCmp }) = ()
```
or maybe we can just documented it well and leave it like this...

Reply:
It's actually only passing `title` and `containerCmp`. We can filter out props when unpacking. I forgot to do it in this component but I usually filter out `overridableUID`.
```js
const Element = ({ overridableUID, ...props }) => {
```